### PR TITLE
add posibility to ignore tests

### PR DESCRIPTION
**Pull request by Alex-Diez**

### CODE REVIEW

File: src/parse.rs [hunk 1]
```diff
@@ -15,8 +15,8 @@ pub trait Parse<Cfg> {
     fn parse(&mut Parser, Cfg) -> Self;
 }
 
-impl Parse<bool> for Test {
-    fn parse(parser: &mut Parser, failing: bool) -> Test {
+impl Parse<(bool, bool)> for Test {
```
Comment (reem) [inline src/parse.rs]: Can you instead wrap these two bools in a simple struct `TestConfig { ignored: bool, failing: bool }` and pass that around instead? I think that will make the code a little clearer and less prone to subtle bugs in the future.

File: tests/ignore.rs [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#![feature(plugin,const_fn)]
+#![plugin(stainless)]
+
+describe! ignored_tests {
+
+    ignore "should be ingored" {
```
Comment (reem) [inline tests/ignore.rs]: typo in `ingored`

File: src/generate.rs [hunk 3]
```diff
@@ -20,13 +20,14 @@ pub trait Generate<Cfg> {
...
     fn generate(self, sp: codemap::Span, cx: &mut base::ExtCtxt, state: &'a DescribeState) -> P<ast::Item> {
-        let Test { description, block, failing } = self;
+        let Test { description, block, failing, ignored } = self;
 
         // Create the #[test] attribute.
         let test_attribute = cx.attribute(sp, cx.meta_word(sp, token::InternedString::new("test")));
 
         // Create the #[should_panic] attribute.
         let should_panic = cx.attribute(sp, cx.meta_word(sp, token::InternedString::new("should_panic")));
+        let ignore = cx.attribute(sp, cx.meta_word(sp, token::InternedString::new("ignore")));
```
Comment (reem) [inline src/generate.rs]: Can you add a comment here saying we are creating the `#[ignore]` attribute?

**Comment by reem** [comment]
Looks good except for the comments I left, let me know when you have a chance to update!

**Comment by Alex-Diez** [comment]
done

File: src/test.rs [hunk 4]
```diff
@@ -6,6 +6,35 @@ use syntax::ast;
 pub struct Test {
     pub description: String,
     pub block: P<ast::Block>,
+    pub failing: bool,
```
Comment (reem) [inline src/test.rs]: this should just embed a `TestConfig` instance

**Comment by reem** [comment]
Left one more comment, should be ready to be squashed and merged after the last comment is addressed.

**Comment by reem** [comment]
Looks great, can you squash and I will merge/release?

**Comment by Alex-Diez** [comment]
Any updates?

**Comment by Alex-Diez** [comment]
Sorry for annoying but are you going to merge this PR?
I am really missing this functionality

**Comment by reem** [comment]
Thanks for bumping. Looks good.

**Pull request merged**